openlibrary/plugins/upstream/addbook.py

```diff
@@ -440,7 +448,8 @@ def save(self, formdata):
 
         saveutil.commit(comment=comment, action="edit-book")
 
-    def new_work(self, edition):
+    @staticmethod
+    def new_work(edition):
```

Comment:
making this `@staticmethod` just makes it different from `new_author` and the other methods below, still seems to work the same, but I'm now thinking making not static and using `self.edition` instead of taking the argument would make more sense.

Is this ever called statically without a `self.edition` set?

Reply:
I switched it to a staticmethod cause I was using it somewhere else temporarily :P But I think staticmethod is better where possible; it's easier to test and to understand (i.e. you _know_ this fn only depends on the edition just from reading the signature, and don't have to ready it's body). I made some more methods static where possible.